package.json

```diff
@@ -27,5 +27,19 @@
     "deep-equal": "~0.2.1",
     "extend": "~2.0.0",
     "pauseable": "~0.1.5"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.11",
+    "jest": "^24.5.0"
+  },
+  "jest": {
```

Comment:
I could've placed these into a separate jest.config.js, but I'd prefer to not bloat the repo's directory. If it were to grow in complexity, and it won't, it'd be appropriate to have a separate config file.